frontend/src/views/applications/ServiceDeliveryView.vue

```diff
@@ -155,12 +155,13 @@ export default {
           await ApplicationService.updateApplication(this.$route.params.applicationGuid, payload)
           await this.getApplication(this.$route.params.applicationGuid)
         }
-        if (this.changedLicences.length > 0) {
-          for (const licence of this.changedLicences) {
-            await LicenceService.updateLicenceDetails(licence.licenceDetailId, licence)
-          }
-          this.changedLicences = []
-        }
+
+        this.changedLicences.forEach(async (licence) => {
+          await LicenceService.updateLicenceDetails(licence.licenceDetailId, licence)
```

Comment:
Hi Jen, I think we should use "await Promise.all", so we can handle requests in parallel.

Reply:
@jenbeckett